backend/src/utils/date.ts

```diff
@@ -1,5 +1,10 @@
-export const daysAgo = (days: number) => {
-  const today = new Date();
-  today.setHours(0, 0, 0, 0);
-  return new Date(today.getTime() - days * 24 * 60 * 60 * 1000);
-};
+const calcDays =
+  ({ future }: { future: boolean }) =>
```

Comment:
why future as object and not just boolean?

Reply:
Because @chenyair likes parameters as objects so that you can send them like in python's positional arguments